.circleci/config.yml

```diff
@@ -87,15 +87,10 @@ aliases:
       name: Installing dependencies with composer
       command: composer update --no-interaction
 
-  - &STEP_COMPOSER_COMPILE
+  - &STEP_COMPOSER_GENERATE
```

Comment:
Since now 'compilation' is in the make file, I preferred to turn to 'generation' for php, as compilation in a context closer to C might be confusing, considering what we do. Not a strong opinion though.